eng/Signing.props

```diff
@@ -1,3 +1,123 @@
 <Project>
-  <!-- TODO: Consolidate the Signing.props files into here. -->
-</Project>
\ No newline at end of file
+
+  <PropertyGroup>
+    <!--
+      Windows arm/arm64 jobs don't have MSIs to sign. Keep it simple: allow not finding any matches
+      here and rely on overall signing validation.
+    -->
+    <AllowEmptySignList>true</AllowEmptySignList>
+  </PropertyGroup>
+
+  <!-- Get artifact locations to sign. -->
+  <Import Project="$(RepositoryEngineeringDir)/liveBuilds.targets" />
+
+  <ItemGroup>
+    <!--
+      Replace the default items to sign with the specific set we want. This allows the build to call
+      Arcade's Sign.proj multiple times for different sets of files as the build progresses.
+    -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+
+    <!-- Find bundle artifacts, which need multiple stages to fully sign. -->
+    <BundleInstallerEngineArtifact Include="$(ArtifactsPackagesDir)**/*engine.exe" />
+    <BundleInstallerExeArtifact Include="$(ArtifactsPackagesDir)**/*.exe" />
+
+    <!-- apphost and comhost template files are not signed, by design. -->
+    <FileSignInfo Include="apphost.exe;comhost.dll" CertificateName="None" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(CrossTargetComponentFolder)' != ''">
+    <CoreCLRCrossTargetItemsToSign Include="$(CoreCLRArtifactsPath)$(CrossTargetComponentFolder)/sharedFramework/*.dll" />
+    <CoreCLRCrossTargetItemsToSign Include="$(CoreCLRArtifactsPath)$(CrossTargetComponentFolder)/sharedFramework/*.exe" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(SignBinaries)' == 'true'">
+    <!-- Sign CoreCLR. -->
+    <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)*.dll" />
+    <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)*.exe" />
+
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)System.Private.CoreLib.dll" />
+
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/crossgen2.exe" />
```

Comment:
Do you expect more `.dlls` in these folders? Should we use a wildcard, or are you doing to prevent signing unnecessary files?

This could leak, that if we add a new assembly we could miss signing it and then break the official build and could be kind of hard to diagnose for people not familiar with signing.

Reply:
Yeah, this folder contains a self-contained publish, so there's a lot of stuff here. It's just a straight port from CoreCLR though: https://github.com/dotnet/coreclr/blob/a9f3fc16483eecfc47fb79c362811d870be02249/eng/Signing.props#L9-L18.

Reply:
I see. Ok, then we can leave as is.